src/components/DataLinks/DataLinks.test.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
-import { act, render, screen, fireEvent } from '@testing-library/react';
-import { DataLinks } from './DataLinks';
-import * as jestFetchMock from 'jest-fetch-mock';
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import { DataLinks } from "./DataLinks";
+import * as jestFetchMock from "jest-fetch-mock";
+import userEvent from "@testing-library/user-event";
 
 jestFetchMock.enableFetchMocks();
 
-describe('DataLinks', () => {
+describe("DataLinks", () => {
```

Comment:
I think some of these lint fixes are incorrect, but it is not relevant to this PR. But definitely something to fix in the next one by using https://github.com/grafana/eslint-config-grafana.